ntbs-service/deployments/int.yml

```diff
@@ -26,6 +26,11 @@ spec:
               secretKeyRef:
                 name: int-audit-db-connection-string
                 key: connectionString
+          - name: ConnectionStrings__reporting
+            valueFrom:
+              secretKeyRef:
+                name: int-reporting-db-connection-string
+                key: connectionString
```

Comment:
Think we need a dev version of this? E.g. in `appsettings.Development.json`

Reply:
That's a good shout! As it's a connection string (with credentials) we'd want to add it to the user secrets.
Alternatively if we want to maintain the ability to dev with no network dependencies, perhaps we add a dev-only context that creates stubs of these functions in a local db (at which point we could just use `appsettings`, too)

Reply:
yes, I am using user secrets, and planning to update localSecrets on azure portal.
Definitely good idea to stub it on local db, otherwise we need to use int-reporting